manifests/charts/istio-control/istio-discovery/templates/poddisruptionbudget.yaml

```diff
@@ -16,7 +16,14 @@ metadata:
     app.kubernetes.io/name: "istiod"
     {{- include "istio.labels" . | nindent 4 }}
 spec:
-  minAvailable: 1
+  {{- if and .Values.minAvailable (not (hasKey .Values "maxUnavailable")) }}
```

Comment:
Lazy question -- some fields in PDB are immutable. are these ones? I don't think it blocks us from modifying them but would be good to at least call out

Reply:
The three settings, minAvailable, maxUnavailable, and unhealthyPodEvictionPolicy are mutable.  I tested out with a 1.33.1 kubectl client.